<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_product</class>
 <widget class="QDialog" name="add_product">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Thêm sản phẩm</string>
  </property>
  <widget class="QLabel" name="lb_title">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Thêm sản phẩm</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>271</width>
     <height>88</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="lb_msp">
        <property name="text">
         <string>Mã sản phẩm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_tensp">
        <property name="text">
         <string>Tên sản phẩm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_danhmuc">
        <property name="text">
         <string>Danh mục</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="le_msp">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_tensp"/>
      </item>
      <item>
       <widget class="QComboBox" name="cb_danhmuc"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>168</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pb_save">
      <property name="text">
       <string>Lưu</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_cancel">
      <property name="text">
       <string>Hủy</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
